code/modules/projectiles/guns/energy.dm

```diff
@@ -150,7 +150,7 @@
 	ammo = /datum/ammo/energy/lasgun/M43
 	ammo_diff = null
 	cell_type = null
-	charge_cost = M43_STANDARD_AMMO_COST
+	charge_cost = TX73_STANDARD_AMMO_COST
```

Comment:
maybe just change it to a generic name instead of changing all the defines every time someone renames this gun

Reply:
I guess I'll go with `ENERGY_STANDARD_AMMO_COST` or perhaps just remove the define? That would go more in line with the whole 'no more defines for weapons'

Reply:
removing the define here is okay but below you should keep it for the OVERCHARGE_AMMO_COST because its unclear what you're checking otherwise